materials/spraymanager/transparent.vmt

```diff
@@ -0,0 +1,8 @@
+"LightmappedGeneric"
+{
+	"$basetexture"	"decals\spraymanager\transparent"
```

Comment:
is it not spraymanager/transparent ? (idk anything about that stuff)

Reply:
That correct, nice catch, updated.